app/posts/apply-for-teacher-training/2022-07-13-improving-the-email-sent-to-candidates-about-new-interviews.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Improving the email sent to candidates about new interviews
+description: We now include the course subject and code. We also show details separately rather than in a single section.
+date: 2022-07-13
+---
```

Comment:
We could link to the pull request as a related item maybe?

Also updated date to today.

```suggestion
date: 2022-08-12
related:
  items:
    - text: Pull request implementing this change
      href: https://github.com/DFE-Digital/apply-for-teacher-training/pull/7276
---
```